### PR TITLE
#11 Fix typo ressources in resources.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: dynawo/sources/Launcher/launcher.in [hunk 1]
```diff
@@ -16,8 +16,8 @@ export BIN_DIR=${DYNAWO_INSTALL_DIR}/@BINDIR_NAME@/
 export SBIN_DIR=${DYNAWO_INSTALL_DIR}/@SBINDIR_NAME@/
 export DDB_DIR=${DYNAWO_INSTALL_DIR}/@DDBDIR_NAME@/
 export SCRIPTS_DIR=${DYNAWO_INSTALL_DIR}/@SBINDIR_NAME@/
-export RESSOURCES_DIR=${DYNAWO_INSTALL_DIR}/@SHAREDIR_NAME@/
-export XSD_DIR=${RESSOURCES_DIR}/xsd/
+export RESOURCES_DIR=${DYNAWO_INSTALL_DIR}/@SHAREDIR_NAME@/
```
Comment (rosiereflo) [inline dynawo/sources/Launcher/launcher.in]: Please update the doc accordingly

File: .gitignore [hunk 2]
```diff
@@ -74,15 +74,15 @@ dynawo/3rdParty/adept/adept-1.1.tar.gz
...
-util/curvesToHtml/ressources/arrow-down.gif
-util/curvesToHtml/ressources/arrow-left.gif
-util/curvesToHtml/ressources/arrow-right.gif
-util/curvesToHtml/ressources/arrow-up.gif
-util/curvesToHtml/ressources/jquery.flot.crosshair.js
-util/curvesToHtml/ressources/jquery.flot.js
-util/curvesToHtml/ressources/jquery.flot.navigate.js
-util/curvesToHtml/ressources/jquery.flot.selection.js
-util/curvesToHtml/ressources/jquery.js
+util/curvesToHtml/resources/arrow-down.gif
```
Comment (rosiereflo) [inline .gitignore]: Did you rename the folder? (if so it seems to be hidden in the review so I prefer to double-check)

File: util/curvesToHtml/csvToHtml/csvToHtml.py [hunk 3]
```diff
@@ -42,13 +42,13 @@ def serie(self):
 
 def readCsvToHtml(csv_file, output_dir, withoutOffset, showpoints):
     full_path = os.path.expanduser(output_dir)
-    # Copy ressources in output directory
-    output_ressources_dir = os.path.join(full_path,"curvesRessources")
+    # Copy resources in output directory
+    output_resources_dir = os.path.join(full_path,"curvesResources")
```
Comment (rosiereflo) [inline util/curvesToHtml/csvToHtml/csvToHtml.py]: Also visible in the documentation

File: .gitignore (hunk 2 above)
Comment (gautierbureau) [reply .gitignore]: No I ill do it

**Pull request merged**